### ISSUE TITLE
modify repo to work with arabic language

### ISSUE THREAD

**Issue by Mohamed209** [astutejoe/tesseractfonts#1]
i decided to make the repo work for languages other than english , i am working on arabic finetuning 
first i modified generate_training_data.sh to be as follows 
`rm -rf train/*
tesseract/src/training/tesstrain.sh --fonts_dir fonts \
	     --fontlist 'DejaVu Sans Mono'\
	     --lang ara \
	     --linedata_only \
	     --langdata_dir langdata_lstm \
	     --tessdata_dir tesseract/tessdata \
	     --save_box_tiff \
	     --maxpages 10 \
	     --output_dir train`
but i got this error 

> === Phase E: Generating lstmf files ===
Using TESSDATA_PREFIX=tesseract/tessdata
[Sun 18 Aug 2019 09:57:53 AM PDT] /usr/bin/tesseract /tmp/ara-2019-08-18.q99/ara.DejaVu_Sans_Mono.exp0.tif /tmp/ara-2019-08-18.q99/ara.DejaVu_Sans_Mono.exp0 --psm 6 lstm.train langdata_lstm/ara/ara.config
Error opening data file tesseract/tessdata/eng.traineddata
Please make sure the TESSDATA_PREFIX environment variable is set to your "tessdata" directory.
Failed loading language 'eng'
Tesseract couldn't load any languages!
Could not initialize tesseract.
ERROR: Program tesseract failed. Abort.
so how to adapt this pipeline for other languages

**Comment by kotebeg** [comment]
yes it is right, but i wonder does it affect when training not English language??

**Comment by sam-kurdi** [comment]
dose it works for Arabic fine-tuning? any sources?
